api/src/main/java/org/pmiops/workbench/institution/InstitutionServiceImpl.java

```diff
@@ -334,6 +368,7 @@ private String generateShortName(String displayName) {
   }
 
   private void validateInstitution(Institution institutionRequest) {
+    // TODO(RW-7027): Figure out what validations we need for tier requirement change.
```

Comment:
That may not well work with my next step plan, i.e. dual write for RT. 
I will add dual write from UI to set RT requirement only. 
CR requirement will be added later.

Reply:
We might be able to set CT to NO_ACCESS. Added some description in https://precisionmedicineinitiative.atlassian.net/browse/RW-7027, this can be added later when we address this todo

Reply:
What I mean is that *this PR* already forces callers to do this - see `testModelToDbError_accessTierNotFound()` so it would be good to fail-fast here as well to make it explicit.

Yes - NO_ACCESS would work.